internal/core/src/common/FieldMeta.cpp

```diff
@@ -20,7 +20,7 @@
 namespace milvus {
 TokenizerParams
```

Comment:
rename method and type name as well: `TokenizerParams` and `ParseTokenizerParams`

Reply:
next pr, all tokenizer in segcore and go code shoud rename to analyzer, but this pr just rename user params.